### PR TITLE
Shuffle edge predict according to edge id mapping

### PR DESCRIPTION
**Pull request by classicsong**
*Issue #, if available:*
#49 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python/graphstorm/model/node_gnn.py [hunk 1]
```diff
@@ -155,6 +155,7 @@ def node_mini_batch_gnn_predict(model, loader, return_label=False):
     -------
     Tensor : GNN prediction results.
     Tensor : GNN embeddings.
+    Tensor : Node index corresponding to prediction results.
```
Comment (zhjwy9343) [inline python/graphstorm/model/node_gnn.py:158]: Do we need to also change edge_gnn.py to add Edge index as a return value?

File: tests/unit-tests/test_gnn.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def require_cache_embed(self):
     pred1, labels1 = node_mini_batch_predict(model, embs, dataloader1, return_label=True)
     dataloader2 = GSgnnNodeDataLoader(data, target_nidx, fanout=[-1, -1],
                                       batch_size=10, device="cuda:0", train_task=False)
-    pred2, emb2, labels2 = node_mini_batch_gnn_predict(model, dataloader2, return_label=True)
+    pred2, _, labels2 = node_mini_batch_gnn_predict(model, dataloader2, return_label=True)
```
Comment (zhjwy9343) [inline tests/unit-tests/test_gnn.py:125]: Do we also need to add edge_mini_batch_gnn_predict() for testing.

File: python/graphstorm/model/node_gnn.py (hunk 1 above)
Comment (classicsong) [reply python/graphstorm/model/node_gnn.py:158]: it does not return node index actually, I forgot to remove the comment.

File: tests/unit-tests/test_gnn.py (hunk 2 above)
Comment (classicsong) [reply tests/unit-tests/test_gnn.py:125]: We do. In L263

File: python/graphstorm/run/gsgnn_ep/ep_infer_gnn.py [hunk 3]
```diff
@@ -71,7 +71,8 @@ def main(args):
     infer.infer(dataloader, save_embed_path=config.save_embed_path,
                 save_prediction_path=config.save_prediction_path,
                 use_mini_batch_infer=config.use_mini_batch_infer,
-                node_id_mapping_file=config.node_id_mapping_file)
+                node_id_mapping_file=config.node_id_mapping_file,
+                edge_id_mapping_file=config.edge_id_mapping_file)
```
Comment (zhjwy9343) [inline python/graphstorm/run/gsgnn_ep/ep_infer_gnn.py:75]: Should users specify these two arguments in yaml or arguments?

Comment (zhjwy9343) [reply python/graphstorm/run/gsgnn_ep/ep_infer_gnn.py:75]: if not, I am fine.

**Review by zhjwy9343 — APPROVED** [review]
LGTM
